### ISSUE TITLE
[Feature Request] Allow use without debrid services...

### ISSUE THREAD

**Issue by kubo6472** [itsToggle/plex_debrid#608]
...instead just call stremio's [server.js](https://dl.strem.io/server/v4.20.3/desktop/server.js) to get a playable mp4/mkv/avi file and start playing that.

**Comment by dreulavelle** [comment]
This app isn't meant to be Stremio. You pair it with an RD account along with Rclone to build a library yourself. 

If you want more of the "on-demand" approach, your better off with Stremio.

**Comment by kubo6472** [comment]
No, I don't think you get me. I'd like to use an alternative to pairing with a debrid service like RD, to try and get files from the internal part that makes stremio "tick", which behaves the same as a debrid service.

**Comment by dreulavelle** [comment]
What kind of alternative? We already scrape the same sources that stremio scrapes. Not sure what your talking about

**Comment by kubo6472** [comment]
Using torrentio to get torrents and then stremio internals to download such torrent. It's like when you won't cache RD torrents, but download them anyway. It I run this script in a DC I can use fast speeds, and download only the parts I'm currently watching.
